lib/omniauth/strategies/wrike.rb

```diff
@@ -7,8 +7,8 @@ class Wrike < OmniAuth::Strategies::OAuth2
 
       option :client_options,
              site: 'https://www.wrike.com/api/v4',
-             authorize_url: 'https://www.wrike.com/oauth2/authorize/v4',
-             token_url: 'https://www.wrike.com/oauth2/token'
+             authorize_url: 'https://www.login.wrike.com/oauth2/authorize/v4',
```

Comment:
[doc link](https://developers.wrike.com/oauth-20-authorization/) states `login.wrike.com` for the host, but shared google doc states `www.login.wrike.com`.  `www.login.wrike.com` doesn't resolve for me